mm2src/mm2_metrics/src/wasm.rs

```diff
@@ -1,5 +1,14 @@
 use super::*;
-use crate::now_ms;
+use common::log::LogWeak;
+
+///  Dummy Labels constructor macro
+#[macro_export]
+macro_rules! mm_label {
+    ($($label_key:expr => $label_val:expr),+) => {{
+         let labels = vec![$(($label_key.to_owned(), $label_val.to_owned())),+];
+         labels
+    }};
```

Comment:
I've just checked that `metrics` and `metrics-util` compile in WASM 🔥 
It seems that now we can use metrics in a browser.
It also means we need to remove the wasm dummy mocks.
But please note that the prometheus doesn't work in WASM (we can't accept incoming connections by listening a server port in a browser).

Reply:
Please also port the [metrics](https://github.com/KomodoPlatform/atomicDEX-API/blob/dev/mm2src/mm2_main/src/rpc/dispatcher/dispatcher_legacy.rs#L91) RPC to WASM since it's supported now.

Reply:
not needed again as the method is already exposed to WASM